publication-rest/src/main/java/no/unit/nva/publication/update/UpdatePublicationHandler.java

```diff
@@ -238,14 +239,27 @@ private Publication unpublishPublication(UnpublishPublicationRequest unpublishPu
                                                             existingPublication,
                                                             userInstance);
         resourceService.unpublishPublication(updatedPublication);
-
+        setPendingTicketsToNotRelevant(updatedPublication);
         updatedPublication = resourceService.getPublication(updatedPublication);
         persistNotification(updatedPublication);
         updateNvaDoi(updatedPublication);
-
         return updatedPublication;
     }
 
+    private void setPendingTicketsToNotRelevant(Publication updatedPublication) {
+        var tickets = resourceService.fetchAllTicketsForResource(Resource.fromPublication(updatedPublication));
+        tickets.filter(this::ticketIsNoLongerRelevant).forEach(this::setTicketToNoLongerRelevant);
+    }
+
+    private void setTicketToNoLongerRelevant(TicketEntry ticket) {
+        ticket.setStatus(TicketStatus.NOT_RELEVANT);
+        ticketService.updateTicket(ticket);
+    }
+
+    private boolean ticketIsNoLongerRelevant(TicketEntry ticket) {
+        return TicketStatus.PENDING.equals(ticket.getStatus());
+    }
```

Comment:
Not sure, but I think that all existing tickets are not longer relevant, otherwise there will be completed tickets in "kurators arbeidsliste" that can point to other publication (when duplicate). Ask Terje :)

Reply:
Bad idea on several levels.

If a record that was previously published with a GeneralSupportticket, how will we know if the support ticket was processed or not?

Also the frontend uses the completed publishing request tickets to show information regarding publishing metadata (who, when and what). If we change that status, then the frontend won't know if the record ever was published or not.

Reply:
I agree if we assume that tickets for unpublished publication are still relevant and should be proceeded. 

Main idea here was:
When curator lists the tasks
And opens the ticket for unpublished publication with duplicate
Ticket will never be shown (we are fetching another publication)

But also that these tickets will never point to original publication (and it is ok, but should they be listed). I think it is more Product owner question.

`Min ikke så fult supre registrering` in `dev` is a very good case when it is confusing.

Reply:
For auditing purposes, the history is important. if the P is revived, we lose the history. Only pending tickets should be marked as irrelevant.

Reply:
If there are requirements that the other tickets shouldn't be shown for certain Ps, then they should be suppressed by FE.